client/js/presenter.js

```diff
@@ -117,9 +136,35 @@ this.subscribeToEvents= function (){
   .on('asq:question_type', function(evt){
     console.log(evt);
     debug.log(evt)
+  })
+  .on('setting:update-live-presentation-settings', updateLivePresentationSettings);
```

Comment:
We need to decide on the naming conventions for the events. Let's have underscores instead of dashes to follow the convention of socket-io events. As far as the settings prefix is concerned, I would replace it either with 'asq' or 'core'. I don't thing settings should have its own namespace, at least not without 10 or more pertinent events. This one is up to you though.